wordle-web/pages/index.vue

```diff
@@ -75,5 +71,37 @@ function onKeyup(event: KeyboardEvent) {
   } else if (event.key.match(/[A-z]/) && event.key.length === 1) {
     game.value?.addLetter(event.key.toUpperCase());
   }
+
+}
+function calcAttempts(){
+  var attempts = 0;
+  if(game.value.gameState == GameState.Won){
+    attempts = game.value.guessIndex + 1;
+  }else{
+    attempts = game.value.guesses.length + 5;
+  }
+  return attempts;
 }
```

Comment:
```suggestion
```
Is this needed? This is being called by `postScore` So at that point, you know the game is being from the watcher placed on it. If you win or lose, the guessIndex should reflect on the number of guesses taken. It's also of note that guesses' length should always be 6 becuase we have to start by filling the guesses with 6 empty words so it can render each word in UI. I would suggest removing this and just directly putting in the `game.value.guessIndex + 1` into `postScore`

Reply:
Agreed, that would be much more elegant.